CATS/Problem/Repository.pm

```diff
@@ -902,13 +902,14 @@ sub log {
 }
 
 sub archive {
-    my ($self, $tree_id) = @_;
+    my ($self, $tree_id, $git_arhive) = @_;
```

Comment:
1) Bad name.  `$use_path`?
2) Why is this parameter needed? It seems that if `repo_path` is non-empty, it should always be used?